### DISCUSSION TITLE
Include decimal extensions

### DISCUSSION THREAD

**Discussion by githubsean** [pawelsalawa/sqlitestudio#4220] [Ideas]
I am writing some code to deal with Ethereum. Ethereum stores values to 18 decimal places - which is more than what the usual numeric handling (for example aggregate functions and addition/subtraction etc) can handle.

There are some "decimal extensions" discussed here: https://www.sqlite.org/floatingpoint.html#decext

These are compiled by default into the sqlite CLI, but not by default into any included source (the amalgamation).

Would it be possible to compile these extensions into sqlitestudio?

**Comment by pawelsalawa** [comment]
I'll consider it, but I'm not making a 100% promise, because I'm not sure yet how problematic (or not) it will be to include it.

**Reply by pawelsalawa** [reply]
#4221 created

**Comment by pawelsalawa** [comment]
Just to let you know - for the upcoming version 3.4.0 this extension will be included (must be explicitly enabled by User in the Extensions Manager window, but it's just few clicks). Actually many more SQLite extensions will be distributed this way:
![image](https://user-images.githubusercontent.com/481846/187045056-ee64802c-ede0-4e38-8f26-2e28669635e8.png)
